website/docs/d/cloud_account.html.markdown

```diff
@@ -4,13 +4,13 @@ layout: "aci"
 page_title: "ACI: aci_cloud_account"
 sidebar_current: "docs-aci-data-source-cloud-account"
 description: |-
-  Data source for ACI Cloud Account
+  Data source for Cloud Network Controller Cloud Account
```

Comment:
Cloud Network Controller Cloud Account sounds confusing
Just a suggestion ->
Data source for the Cloud Account